concert/cloudformation.yml

```diff
@@ -383,6 +386,117 @@ Resources:
         - DynamoDBCrudPolicy:
             TableName: !Ref Table
 
+  AggregateChoiceDataStream:
+    Type: AWS::KinesisFirehose::DeliveryStream
+    Properties:
+      DeliveryStreamName: !Sub ${Environment}-AggregateChoiceDataStream
+      S3DestinationConfiguration:
+        BucketARN: !GetAtt AggregateChoiceDataS3Bucket.Arn
+        BufferingHints:
+          IntervalInSeconds: 60
+          SizeInMBs: 50
+        CompressionFormat: UNCOMPRESSED
+        RoleARN: !GetAtt AggregateChoiceDataStreamRole.Arn
+        CloudWatchLoggingOptions:
+          Enabled: true
+          LogGroupName: !Sub ${Environment}-enchanted-brain-aggregateChoiceDataStreamLogs
+          LogStreamName: !Sub ${Environment}-enchanted-brain-aggregateChoiceDataStreamLogs
+
+  AggregateChoiceDataStreamLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub ${Environment}-enchanted-brain-aggregateChoiceDataStreamLogs
+      RetentionInDays: 30
+  AggregateChoiceDataStreamLogStream:
+    Type: AWS::Logs::LogStream
+    Properties:
+      LogGroupName: !Sub ${Environment}-enchanted-brain-aggregateChoiceDataStreamLogs
+      LogStreamName: s3Backup
+
+  AggregateChoiceDataStreamRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: firehose.amazonaws.com
+            Action: sts:AssumeRole
+      Policies:
+        - PolicyName: firehose_delivery_policy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action: s3:*
+                Resource: '*'
+              - Effect: Allow
+                Action: logs:PutLogEvents
+                Resource: !Sub arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/${Environment}-enchanted-brain-aggregateChoiceDataStreamLogs:*
```

Comment:
You can use `!GetAtt AggregateChoiceDataStreamLogGroup.Arn` to simplify this and avoid typos